src/dom.js

```diff
@@ -936,3 +936,12 @@ export function domOrderComparator(element1, element2) {
   // if fe2 is following or contained by fe1, then fe1 is before fe2
   return -1;
 }
+
+/**
+ * Returns true if the document is an amp-story.
+ * @param {!Document} doc
+ * @return {boolean}
+ */
+export function isStoryDescendant(doc) {
```

Comment:
FYI @alanorozco added a [util](https://github.com/ampproject/amphtml/blob/105646dc782d95792573a2a614ef9701d2d3775f/src/utils/story.js#L28-L30) for this in #19475